.config/nvim/lua/codecompanion/_extensions/copilot_adapter/init.lua

```diff
@@ -124,7 +124,7 @@ local function process_prompt_file(file_path, opts)
   -- Handle VS Code custom chat modes vs prompts
   -- VS Code uses file extensions: .chatmode.md for modes, .prompt.md for prompts
   local is_mode = is_chatmode_file(file_path)
-  if is_mode and not opts.enable_modes then
+  if is_mode and not opts.enable_chatmodes then
```

Comment:
please also here "is_automode"

real rename everything automode related, which is currently named only mode to automode